include/openmc/mesh.h

```diff
@@ -946,6 +946,7 @@ class LibMesh : public UnstructuredMesh {
   LibMesh(pugi::xml_node node);
   LibMesh(const std::string& filename, double length_multiplier = 1.0);
   LibMesh(libMesh::MeshBase& input_mesh, double length_multiplier = 1.0);
+  LibMesh(libMesh::MeshBase& input_mesh,const std::string& extra_element_integer_name, double length_multiplier = 1.0);
```

Comment:
Another similar change... and in the cpp file
```suggestion
  LibMesh(libMesh::MeshBase& input_mesh,const std::string& cluster_element_integer_name, double length_multiplier = 1.0);
```